vagrant/base.sh

```diff
@@ -112,9 +112,9 @@ get_kafka 0.10.2.1 2.11
 chmod a+rw /opt/kafka-0.10.2.1
 get_kafka 0.11.0.2 2.11
 chmod a+rw /opt/kafka-0.11.0.2
-get_kafka 1.0.0 2.11
+get_kafka 1.0.0 2.12
 chmod a+rw /opt/kafka-1.0.0
-get_kafka 1.0.1 2.11
+get_kafka 1.0.1 2.12
```

Comment:
we can also add 1.1 here.

Reply:
We cannot. Need to downgrade to 2.11 anyway. Also, we can only pick one Scala version as there is only one directory for the Kafka version. We would need to change a lot of stuff to encode the Scala version in the directory(structure) to support different version. Even if we want to do this, it would be a separate PR.

Reply:
I didn't quite follow this. If you're saying that switching to scala 2.12 caused the runtime error about class version, that sounds about right.

But does this mean that we cannot add Kafka 1.1.x?

Reply:
Sorry for the confusion -- my bad. We an add 1.1.x -- will update this PR after I merged the 1.1 PR and cherry-picked those latest changes.